src/middleware/queries/externalAgents.ts

```diff
@@ -122,6 +122,6 @@ export function tickerExternalAgentActionsQuery(
 
   return {
     query,
-    variables: { ...filters, size: size?.toNumber(), start: start?.toNumber() },
+    variables: { ...filters, ...getSizeAndOffset(size, start) },
```

Comment:
why not always remove undefined here? does it have to do with having all the args produced internally vs user provided?